src/scripts/widget.js

```diff
@@ -0,0 +1,57 @@
+---
+---
+$(function() {
+    var WIDGET_DISPLAYED_FLAG = 'widget-displayed';
+
+    function sendGaEvent(event) {
+        {% if jekyll.environment == "production" %}
+
+        ga('send', 'event', 'widget', event);
+
+        {% endif %}
+    }
+
+    function updateWidgetVisibility() {
+        if(window.needShowShareWidget()) {

```

Comment:
We need to. For example, we need to check if we are in the documentation and not in Getting Started